packages/xo-web/src/xo-app/vm/tab-advanced.js

```diff
@@ -487,6 +487,8 @@ export default class TabAdvanced extends Component {
     return deleteVtpm(vtpm)
   }
 
+  _updateUser = user => editVm(this.props.vm, { creation: { ...this.props.vm.creation, user: user.id } })
```

Comment:
`user.set` only accept `creation.user` as parameters.
```suggestion
  _updateUser = user => editVm(this.props.vm, { creation: { user: user.id } })
```